README.md

```diff
@@ -187,12 +187,12 @@ In this release, we focused on releasing new features and improvements.
 
 For [v0.65][github-next-release-work], we'll work on below:
 
-- Ship .NET self contained and shared between utilities
-- Stability / bug fixes
+- Ship .NET self-contained and shared between utilities
+- Stability/bug fixes
```

Comment:
Space between slash is intentional

Reply:
True, on more then one occasion. Might be an indication that a different writing style would be even better, though. Yet, low prio.